packages/fscomponents/src/components/FilterList/FilterListDrilldown.tsx

```diff
@@ -347,6 +351,9 @@ export class FilterListDrilldown extends PureComponent<
         <TouchableOpacity
           style={S.secondLevelHeader}
           onPress={this.backToFirstLevel}
+          accessibilityRole={'button'}
+          accessibilityHint={'Go back one filter level'}
```

Comment:
It seems like this and "Toggles Filter" above should be in the translation file, too.